src/System.Management.Automation/engine/runtime/CompiledScriptBlock.cs

```diff
@@ -1520,11 +1520,7 @@ private static bool WriteScriptBlockToLog(ScriptBlock scriptBlock, int segment,
 
```

Comment:
@TravisEz13, your last commit had 2 failures in `PowerShell-CI-linux`
Basic SysLog tests on Linux.Verifies scriptblock logging
```powershell
Expected regular expression 'Creating Scriptblock text \(1 of 1\):#012\$pid[?\?]& \(\[scriptblock\]::create\("Write\-Verbose 'testheader123' ;Write\-verbose 'after'"\)\)(#012)*ScriptBlock ID: [0-9a-z\-]*#012Path:.*' to match 'Creating Scriptblock text (1 of 1):#012$pid?& ([scriptblock]::create("Write-Verbose 'testheader123' ;Write-verbose 'after'"))?#012#012ScriptBlock ID: 263aebb2-a3e9-4f37-8121-7c4b992ce77d#012Path: /tmp/9c51aa89-25d4-4e87-a6fe-a717fe13d7fb/test01.ps1', but it did not match.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Logging.Tests.ps1: line 215
215:         $createdEvents[1].Message | Should -Match ($scriptBlockCreatedRegExTemplate -f (Get-RegEx -SimpleMatch $Script.Replace([System.Environment]::NewLine,"`u{23CE}")))
```
Basic SysLog tests on Linux.Verifies scriptblock logging with null character
```powershell
Expected regular expression 'Creating Scriptblock text \(1 of 1\):#012\$pid[?\?]& \(\[scriptblock\]::create\("Write\-Verbose 'testheader123\$\(\[char\]0x0000\)' ;Write\-verbose 'after'"\)\)(#012)*ScriptBlock ID: [0-9a-z\-]*#012Path:.*' to match 'Creating Scriptblock text (1 of 1):#012$pid?& ([scriptblock]::create("Write-Verbose 'testheader123$([char]0x0000)' ;Write-verbose 'after'"))?#012#012ScriptBlock ID: 59c2e14c-c82f-47ca-9795-f485c2379cb7#012Path: /tmp/9c51aa89-25d4-4e87-a6fe-a717fe13d7fb/test01.ps1', but it did not match.
at <ScriptBlock>, /home/vsts/work/1/s/test/powershell/Host/Logging.Tests.ps1: line 244
244:         $createdEvents[1].Message | Should -Match ($scriptBlockCreatedRegExTemplate -f (Get-RegEx -SimpleMatch $Script.Replace([System.Environment]::NewLine,"`u{23CE}")))
```